config/prod/kelsey-windows.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision a Windows EC2 instance
+template_path: remote/managed-ec2-win.yaml
+stack_name: kelsey-windows
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
```

Comment:
Shouldn’t  this be something specific to sysbio?

Reply:
It will be used across consortia so I left it vague...